types/react-native-material-dropdown/index.d.ts

```diff
@@ -0,0 +1,157 @@
+// Type definitions for react-native-material-dropdown 0.11
+// Project: https://github.com/n4kz/react-native-material-dropdown
+// Definitions by: Jaydeep <https://github.com/jaydeep987>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
Is there a particular reason you need a minimum version of 3.0? It seems like you'd be fine with 2.8.